GVRf/Framework/src/org/gearvrf/GVRLight.java

```diff
@@ -29,8 +29,49 @@ public GVRLight(GVRContext gvrContext) {
         setDiffuseIntensity(1.0f, 1.0f, 1.0f, 1.0f);
         setSpecularIntensity(1.0f, 1.0f, 1.0f, 1.0f);
         isEnabled = true;
+        name="";       
+        this.lightType=GVRLightType.GVRLightSource_SPOT;
     }
 
+    
+    /*
+     *  Get the {@code light_name}
+     *  this is to make the parameters to be same as that of assimp's aiLight structure.
+     *  
+     *  @return The name of light source
+     */
+    public String getLightName(){
+    	return this.name;
+    }
+    /*
+     *  Set the name of light
+     *  It is to allow the user to set the name of  light manually from the model file
+     *  
+     *  @param name: name of light
+     */
+
+    public void SetLightName(String name){
```

Comment:
setLightName instead of SetLightName.

You might want to run the Java formatter in Eclipse; first make sure you've imported the Android formatter (https://github.com/android/platform_development/blob/master/ide/eclipse/android-formatting.xml).